internal/pkg/manifest/env.go

```diff
@@ -108,7 +108,7 @@ type subnetConfiguration struct {
 }
 
 type environmentObservability struct {
-	ContainerInsights *bool `yaml:"container_insights"`
+	ContainerInsights *bool `yaml:"container_insights,omitempty"`
```

Comment:
dumb question: what fields to `omitempty` and what fields not to?

Reply:
from playing around with it, we need it for everything. Otherwise, the field is rendered with the zero-value `null`, `0`, `""`...

Reply:
What about `subnet.ID` and `subnet.CIDR` 🤔 ?

Reply:
I'm a dummy and will include in the next PR